src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonTestHelper.cs

```diff
@@ -753,23 +753,5 @@ public static void AssertContentsNotEqualAgainstJsonNet(string expectedValue, st
 
             throw ex is null ? new ThrowsException(typeof(TException)) : new ThrowsException(typeof(TException), ex);
         }
-
-#if NETCOREAPP
```

Comment:
Ah, I see you just moved existing code. Like I said I think you could just remove this and use `JsonTestHelpers.AssertJsonEqual` for more robust assertion checking.